frontend/js/src/album/AlbumPage.tsx

```diff
@@ -520,33 +519,19 @@ export default function AlbumPage(): JSX.Element {
                 More on CritiqueBrainz…
               </a>
             </>
-          ) : (
-            <p>Be the first to review this album on CritiqueBrainz</p>
-          )}
-          <button
-            type="button"
-            className="btn btn-info"
-            data-toggle="modal"
-            data-target="#CBReviewModal"
-            onClick={() => {
-              NiceModal.show(CBReviewModal, {
-                entityToReview: [
-                  {
-                    type: "release_group",
-                    mbid: albumMBID,
-                    name: album.name,
-                  },
-                  {
-                    type: "artist",
-                    mbid: artist.artists[0].artist_mbid,
-                    name: artist.artists[0].name,
-                  },
-                ],
-              });
+          ) : null}
+          <CBReview
```

Comment:
I think it would look better if we had reviews to one side and the form to the other side, something like this:
![image](https://github.com/user-attachments/assets/eaab714e-5935-4e99-b200-912d46407ea7)
Using bootstrap columns I guess.
I had to muck about with the right-hand side reviews margin a bit to remove the top margin so it all aligns.

And for mobile view, I guess something like that? 
![image](https://github.com/user-attachments/assets/e4b860cb-0274-4252-a492-ecbcb14f982c)

Reply:
Cool! This is how it looks right now:

<img width="1123" alt="image" src="https://github.com/user-attachments/assets/e2c15d76-e25d-4aaf-a666-8d2d39d44327" />

And on mobile view: 
<img width="536" alt="image" src="https://github.com/user-attachments/assets/32785d2c-9506-4dc1-b5da-ac9b5f9d5949" />